src/ray/core_worker/core_worker_process.cc

```diff
@@ -344,9 +344,9 @@ CoreWorker &CoreWorkerProcessImpl::GetCoreWorkerForCurrentThread() {
       // In this case, we should exit without crashing.
       // TODO (scv119): A better solution could be returning error code
       // and handling it at language frontend.
-      RAY_LOG(ERROR) << "The global worker has already been shutdown. This happens when "
-                        "the language frontend accesses the Ray's worker after it is "
-                        "shutdown. The process will exit";
+      RAY_LOG(INFO) << "The global worker has already been shutdown. This happens when "
```

Comment:
Can you keep it ERROR for driver case? I think otherwise, it can have surprising behaviors like driver exits without any error

Reply:
Updated.